validator/distro_validator.go

```diff
@@ -547,6 +547,20 @@ func ensureHasValidVirtualWorkstationSettings(ctx context.Context, d *distro.Dis
 			Level:   Error,
 		})
 	}
+	linuxDistros := []string{
+		evergreen.ArchLinux386,
+		evergreen.ArchLinuxPpc64le,
+		evergreen.ArchLinuxS390x,
+		evergreen.ArchLinuxArm64,
+		evergreen.ArchLinuxAmd64,
+	}
+
+	if !utility.StringSliceContains(linuxDistros, d.Arch) {
+		errs = append(errs, ValidationError{
+			Message: "spawnable distros must use Linux architecture",
```

Comment:
should this say `workstation distros`? some windows distros are user spawnable on prod